src/diffusers/loaders.py

```diff
@@ -180,6 +180,7 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
         subfolder = kwargs.pop("subfolder", None)
         weight_name = kwargs.pop("weight_name", None)
         use_safetensors = kwargs.pop("use_safetensors", None)
+        network_alpha = kwargs.pop("network_alpha", None)
```

Comment:
this needs documentation I think

Reply:
This is about the comment. https://github.com/huggingface/diffusers/pull/3437#discussion_r1195294967 Now that we have an English translation of the document, how about a comment like this?

```suggestion
        # This value is set during training, equivalent to `--network_alpha`
        # See https://github.com/darkstorm2150/sd-scripts/blob/main/docs/train_network_README-en.md#execute-learning
        network_alpha = kwargs.pop("network_alpha", None)
```

Reply:
Works for me!

Reply:
Same!